### PR TITLE
Fix BaseArrayHelper::htmlDecode()

### PR DESCRIPTION
**Pull request by WinterSilence**
Add missed second argument on recursive calling.

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

**Comment by WinterSilence** [comment]
tests updated

File: tests/framework/helpers/ArrayHelperTest.php [hunk 1]
```diff
@@ -1184,30 +1184,33 @@ public function testHtmlDecode()
             'cde' => false,
             3 => 'blank',
             [
-                '<>' => 'a&lt;&gt;b',
```
Comment (bizley) [inline tests/framework/helpers/ArrayHelperTest.php:1187]: Is there a reason to remove that case?

Comment (WinterSilence) [reply tests/framework/helpers/ArrayHelperTest.php:1187]: изображение

Comment (bizley) [reply tests/framework/helpers/ArrayHelperTest.php:1187]: Yes, we can keep it as another case to test.

Comment (WinterSilence) [reply tests/framework/helpers/ArrayHelperTest.php:1187]: @bizley it's invalid key - i fixed recursive decoding

Comment (bizley) [reply tests/framework/helpers/ArrayHelperTest.php:1187]: Don't we want to test what is happening when key or value is already encoded in the first place?

Comment (WinterSilence) [reply tests/framework/helpers/ArrayHelperTest.php:1187]: rollback

**Review by bizley — APPROVED** [review]

**Review by Arhell — APPROVED** [review]

**Comment by bizley** [comment]
Please add a changelog line, thanks.

**Comment by bizley** [comment]
👍🏻

**Pull request merged**